### DISCUSSION TITLE
Missing io.grpc.health.v1.DubboHealthTriple

### DISCUSSION THREAD

**Discussion by sanshi-07** [apache/dubbo#13826] [Question]
本地启动dubbo测试，protobuf编译完后也没看io.grpc.health.v1.DubboHealthTriple、io.grpc.health.v1.Health两个类

**Comment by AlbumenJ** [comment]
dubbo-rpc/dubbo-rpc-triple/src/main/proto/health.proto

**Comment by sanshi-07** [comment]
忽略这个，已经解决
